src/dmd/mars.d

```diff
@@ -70,6 +70,25 @@ private void logo()
     printf("DMD%llu D Compiler %s\n%s %s\n", cast(ulong)size_t.sizeof * 8, global._version, global.copyright, global.written);
 }
 
+/**
+Print DMD's logo with more debug information and error-reporting pointers.
+
+Params:
+    stream = output stream to print the information on
+*/
+extern(C) void internalFailureLogo(FILE* stream)
+{
+    stream.fprintf("---\n");
+    stream.fprintf("ERROR: This is a compiler bug.\n" ~
+            "Please report it via https://issues.dlang.org/enter_bug.cgi\n" ~
+            "with, preferably, a reduced, reproducible example and the information below.\n");
+    stream.fprintf("DustMite (https://github.com/CyberShadow/DustMite/wiki) can help with the reduction.\n");
```

Comment:
Time to make dustmite a dlang project? 😉

Reply:
Yeah :+1: 
FYI: it gets checked out as a "full source" to the tools repo already (https://github.com/dlang/tools/tree/master/DustMite), which is a bit useless as no one is developing on this copy.

CC @andralex